### PR TITLE
contracts end email not send

### PR DESCRIPTION
**Pull request by yusuffGoodie**
## Is this a Feature, Chore or Bug?
- [] Feature
- [] Chore
- [] Bug

## Analysis and design (optional)
Analyse and attach the design documentation

## Solution description
Describe your code changes in detail for reviewers.

## Is there a business logic within a doctype?
    - [] Yes
    - [] No

## Output screenshots (optional)
Post the output screenshots, if a UI is affected or added due to this feature.

## Areas affected and ensured
List out the areas affected by your code changes.

## Is there any existing behavior change of other features due to this code change?
Mention Yes or No. If Yes, provide the appropriate explanation.

## Did you test with the following dataset?
- [] Existing Data
- [] New Data

## Was child table created?
    - [] is attachment required?
        did you test attachment
## Did you delete custom field?
    - [] Yes
    - [] No
        If yes, did you write a delete patch?

## Is patch required?
- [] Yes
- [] No
    ## Was the patch test?

## Which browser(s) did you use for testing?
  - [] Chrome
  - [] Safari
  - [] Firefox

### CODE REVIEW

File: one_fm/operations/doctype/contracts/contracts.py [hunk 1]
```diff
@@ -1028,56 +1028,58 @@ def send_contract_reminders(is_scheduled_event=True):
     Args:
         is_scheduled_event -> Boolean (Default True) If method is triggered from anywhere else than the scheduled event, Pass "False" to avoid email trigger check from "ONEFM General Setting"
     """
```
Comment (NoBoneZ) [inline one_fm/operations/doctype/contracts/contracts.py:1030]: Please make the PR to staging

File: one_fm/operations/doctype/contracts/contracts.py [hunk 2]
```diff
@@ -1028,56 +1028,58 @@ def send_contract_reminders(is_scheduled_event=True):
...
-                       "contract_termination_decision_period": each[0],
-                       "contract_termination_decision_date": each[2],
-                       "end_date": each[6],
-                       "duration": each[7],
-                       "document_id": each[3],
-                       "link": frappe.utils.get_url_to_form("Contracts",each[3]),
-                       "attachment": frappe.utils.get_url(each[10]) if each[10] else None}
-            msg = frappe.render_template('one_fm/templates/emails/contracts_reminder.html', context=context)
-            sendemail(recipients=[users], subject="Expiring Contracts", content=msg,is_scheduler_email=is_scheduled_event)
+    try:
```
Comment (NoBoneZ) [inline one_fm/operations/doctype/contracts/contracts.py:1031]: Please what change did you make

Comment (yusuffGoodie) [reply one_fm/operations/doctype/contracts/contracts.py:1031]: I wrap everything in try and catch and I also pass users variable into the sendemail function directly

File: one_fm/operations/doctype/contracts/contracts.py (hunk 1 above)
Comment (yusuffGoodie) [reply one_fm/operations/doctype/contracts/contracts.py:1030]: Okay

**Review by NoBoneZ — APPROVED** [review]

**Pull request merged**